knowledge-base/handling-errors-occurring-in-a-custom-resolver-used-by-a-reporting-rest-service.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Handling errors occurring in a Custom Resolver used by a Reporting REST Service
+description: Handling errors occurring in a Custom Resolver used by a Reporting REST Service.
+type: how-to
+page_title: Handling errors occurring in a Custom Resolver used by a Reporting REST Service
+slug: handling-errors-occurring-in-a-custom-resolver-used-by-a-reporting-rest-service
+res_type: kb
+---
+
+## Description
+
+Handling errors occurring in a Custom Resolver used by a Reporting REST Service  
```

Comment:
Provide a bit more details to the description. For example, in what scenarios this is useful and how it helps.